pages/_app.js

```diff
@@ -1,9 +1,50 @@
+import { ThirdwebProvider, ChainId } from "@thirdweb-dev/react";
+import '@rainbow-me/rainbowkit/styles.css';
+import { darkTheme, getDefaultWallets, RainbowKitProvider } from "@rainbow-me/rainbowkit";
+import { chain, configureChains, createClient, WagmiConfig, } from 'wagmi';
+import { alchemyProvider } from 'wagmi/providers/alchemy';
+import { publicProvider } from 'wagmi/providers/public';
 import "../index.css"
+import Toaster from "../components/Toaster/Toaster"
+import { ThemeProvider } from "styled-components";
+import { theme } from "../theme/theme";
```

Comment:
Please, order the imports. I recommend to order by import from `module`, internal components and last `.css` files

Reply:
u are right